ci/lib/install_functions.inc.sh

```diff
@@ -522,6 +526,17 @@ linux_install_debian() {
     "${build_dependencies_deb[@]}" \
     "$@"
 
+  if [ "${CC-gcc}" = "clang" ]; then
```

Comment:
Same issue as above --- with `$CC` set to empty string, `"${CC-gcc}"` would still be empty string.  Is this intended?

Reply:
CC="" will force cmake default which would be gcc. If CC is unset it will also force cmake default but it may fail at this line depending on shell settings.